example/standalone_git/main.tf

```diff
@@ -0,0 +1,34 @@
+module "nifi_registry" {
+  source = "../.."
+
+  # nomad
+  nomad_datacenters = ["dc1"]
+  nomad_namespace   = "default"
+
+
+  # nifi
+  service_name    = "nifi-registry"
+  host            = "127.0.0.1"
+  port            = 18080
+  container_image = "michalklempa/nifi-registry:0.8.0"
+  mode            = "git"
```

Comment:
I suggest that `mode` have the same value overall, in this case `mode=standalone_git`

Reply:
I have changed `mode = standalone_git`